### PR TITLE
update dependencies versions

### PR DESCRIPTION
**Pull request by RegNex**
Animation package `animations: ^1.1.0` has some [issue](https://github.com/flutter/flutter/pull/62317) with updated flutter `master branch 1.21.0-6.0.pre.117` . This has been fix on `animations: ^1.1.2`.

### CODE REVIEW

File: example/lib/app.dart [hunk 1]
```diff
@@ -15,8 +15,6 @@ class App extends StatelessWidget {
     return MaterialApp(
       title: title,
       home: const HomePage(title: title),
-      theme: lightTheme(),
-      darkTheme: darkTheme(),
```
Comment (mono0926) [inline example/lib/app.dart:19]: Why did you delete this?
I'd like to keep it.

Comment (RegNex) [reply example/lib/app.dart:19]: There was no reference to it in the code.

Comment (mono0926) [reply example/lib/app.dart:19]: I can’t merge with this change.

Comment (RegNex) [reply example/lib/app.dart]: changes made.
